lib/flowViewport/sceneIndex/fvpIsolateSelectSceneIndex.cpp

```diff
@@ -179,6 +188,20 @@ HdSceneIndexPrim IsolateSelectSceneIndex::GetPrim(const SdfPath& primPath) const
     // HYDRA-1242: setting visibility on GeomSubset prim causes hang in Hydra
     // Storm.
     if (!included && !isGeomSubset(inputPrim)) {
+        // If an instancer is not included, none of its instances are, so set
```

Comment:
Mark all instances as invisible by using the instancer mask.  We still set the instancer itself as invisible, even though for Hydra Storm this does not seem to have any effect.